public/index.html.erb

```diff
@@ -75,7 +75,7 @@
         <div class="col-lg-4 col-md-4">
           <h4 class="bold-headline"><i class="fa fa-quote-left icons"></i>Feedback</h4>
           <div class="teal-line"></div>
-          <p class="small-text">"It's great to see the power of Slack's real-time nature applied to such a real-time issue. I hope this will inspire the development of more tech-for-good!"<br>
+          <p class="small-text"><span class="bold">"</span>It's great to see the power of Slack's real-time nature applied to such a real-time issue. I hope this will inspire the development of more tech-for-good!<span class="bold">"</span><br>
```

Comment:
The quote should be `&quot;` though. Just because it works in some browsers doesn't make it valid HTML.